Automattic/newspack-newsletters
# Update posts inserter defaults

### All Submissions:






### Changes proposed in this Pull Request:

This updates the default settings of the Posts Inserter block:

* Date is now hidden
* Excerpt is displayed and quite short (15 words)

Looking at the different layouts from Dan, the excerpt is far more important than the date.

### How to test the changes in this Pull Request:

1. Add the inserter posts, check the default settings
2. Switch to this branch
3. Add another inserter posts, check the default settings now

### Other information:
